test/e2e/pkg/grammar/clean-start/abci_grammar_clean_start.md

```diff
@@ -0,0 +1,33 @@
+```
+package "github.com/cometbft/cometbft/test/e2e/pkg/grammar/clean-start"
+
```

Comment:
Is it possible to add a comment with a link to the original grammar? (same for the recovery grammar)